pallets/staking/src/lib.rs

```diff
@@ -2574,11 +2850,7 @@ where
             }
         }
 
-        Ok(consumed_weight)
-    }
-
-    fn can_report() -> bool {
```

Comment:
The entry check used for on_offence() has now been changed to: if !Self::era_election_status().is_closed(), is there a problem here?

Reply:
`fn can_report() -> bool {
		Self::era_election_status().is_closed()
}`

There are the same meaning